### PR TITLE
Adds pattern shape for SemVer strings

### PR DESCRIPTION
**Pull request by zvr**
Signed-off-by: Alexios Zavras (zvr) <github@zvr.gr>

### CODE REVIEW

File: model/Core/Classes/SemVer.md [hunk 1]
```diff
@@ -4,11 +4,12 @@ SPDX-License-Identifier: Community-Spec-1.0
...
 ## Summary
 
-A String constrained to the SemVer 2.0.0 specification.
+A string constrained to the SemVer 2.0.0 specification.
 
 ## Description
 
-The semantic version is a String constrained to the SemVer 2.0.0 specification.
+The semantic version is a string
+that is following the specification of Semntic Versioning 2.0.0.
```
Comment (kestewart) [inline model/Core/Classes/SemVer.md:12]: Please insert link to Semantic Versioniong 2.0.0 reference.

**Review by kestewart — CHANGES_REQUESTED** [review]

**Comment by maxhbr** [comment]
This adds a new feature which needs to be added to https://github.com/spdx/spec-parser/ ?

File: model/Core/Classes/SemVer.md (hunk 1 above)
Comment (maxhbr) [reply model/Core/Classes/SemVer.md:12]: probably

```suggestion
that is following the specification of [Semntic Versioning 2.0.0](https://semver.org/).
```

**Comment by zvr** [comment]
@maxhbr yes, I've already opened https://github.com/spdx/spec-parser/issues/47 for this

**Review by kestewart — APPROVED** [review]

**Comment by kestewart** [comment]
Thanks for the update to add the link.

**Pull request merged**